rg_shipit/views/templates/admin/configure-bottom.tpl

```diff
@@ -30,3 +37,12 @@
         </ul>
     </p>
 </div>
+
+<div class="panel">
+    <h3><i class="icon icon-tags"></i> Log de Errores</h3>
+    <p>     
```

Comment:
Why usin a paragraph and no a new div?

Reply:
p is usually used for give it a _new line_ behavior but, what you both are missing here is that list (`ul`) take the same inherit behavior so, no div and no paragraph needed if that's the case here.

Reply:
though `p` is fine and I like it most than a `div` with a custom class that must give it padding and a width of 100% at least to have the same UI effect

Reply:
i kept the design, legacy code